### ISSUE TITLE
How to train Graphormer for autoregressive task?

### ISSUE THREAD

**Issue by rifatarefin** [microsoft/Graphormer#107]
Can we train Graphormer to generate graphs, like what the GPT models do in text generation? Can we apply attention masks similar to the GPT models?

**Comment by zhengsx** [comment]
Yes from the implementation perspective (you could refer to the auto-regressive implementation in Fairseq), but the strategy of 
 generating graph should be carefully designed, which is not easy as sequence generation.
